kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -500,7 +500,7 @@ abstract class TFrontendService(name: String)
 
   private def notSupportTokenErrorStatus = {
     val errorStatus = new TStatus(TStatusCode.ERROR_STATUS)
-    errorStatus.setErrorMessage("Delegation token is not supported")
+    errorStatus.setErrorMessage(DELEGATION_TOKEN_IS_NOT_SUPPORTED)
```

Comment:
Is this change necessary?

Reply:
> Is this change necessary? Or you can define it at test side

I need to judge whether the Delegation token is not supported in the sendCredentials method of KyuubiSyncThriftClient.scala, if not set I need to use this string multiple times